pipeline/flows/matching.py

```diff
@@ -18,6 +18,7 @@
     extract_recent_matches,
     extract_users,
 )
+from pipeline.load.display_metrics import display_internal_matching_metrics
```

Comment:
Don't forget to import the compute metrics transform, you need to compute the metrics before you can display them.